fendermint/app/src/settings.rs

```diff
@@ -58,7 +59,21 @@ pub struct FvmSettings {
 pub struct IPCSettings {
     #[serde(deserialize_with = "deserialize_subnet_id")]
     pub subnet_id: SubnetID,
-    pub config: fendermint_vm_topdown::Config,
+    /// The config for top down checkpoint. It's None if subnet id is root or not activating
+    /// any top down checkpoint related operations
+    pub topdown: Option<fendermint_vm_topdown::Config>,
```

Comment:
I reckon it's never going to be `None` because it's in the `default.toml` file.

Reply:
FYI what I started doing in https://github.com/consensus-shipyard/fendermint/pull/210 is to put empty values into fields that have no reasonable defaults, and put some test values into `test.toml` so I can make sure the deserialization works. 

But I think in this case we can leave the defaults like delays etc. in there if you want. 

Another option is that there is a `mode` parameter on the CLI which can be used to overlay config files on top of the `default.toml` file, so you could have an `lotus-parent.toml` variant that sets the delay to 900, but I think this is such a blunt tool we should not use it like this and rely on env vars and custom config files instead that we put in there at the time of deployment.

Reply:
I removed them from `default.toml`, I assume not everyone wants to start `ipc` in the first place. Adding to `test.toml` seems to be a better idea.

Reply:
Maybe add it to the `default.toml` but commented out, so people can still discover these and don't have to guess the syntax. You can also provide comments above them, which is useful.

In a similar situation with the IPLD Resolver I decided to include it, so people don't have to remember the dozens of fields that need to be filled in if they want to use it, with suitable default values, but deduct when it's disabled based on the absence of fields which don't have reasonable defaults. That way all they have to do is provide that field in an env var, and they are done.